### DISCUSSION TITLE
profile shortcuts and imperial units

### DISCUSSION THREAD

**Discussion by doublejj0321** [SauceLLC/sauce4zwift#62] [General]
Is it possible to create a keyboard shortcut to change between the different profiles? I use one profile for running and one for riding .
Possible to change metric and imperial from the gauges itself; example I would like to see a the pace gauge in KMH and MPH. basically having two gauges sitting next to each other showing the two speeds. Unless I'm missing something this setting is located under the general settings which changes everything to imperial.

Awesome program BTW!!!!

**Comment by mayfield** [comment]
I don't have global shortcuts yet but if/when that feature is added it should be possible to change profiles with one.  Re locale units, you're correct it's global at present.  I'll make of note of your feature request.
